posts/inteview-with-clever-tykes.md

```diff
@@ -0,0 +1,64 @@
+**Hello. So we're chatting because you've written a series of storybooks for children that we think will be of particular interest to Exericsm's users - especially those who are parents. Would you give a quick overview of them?**
+
+Yes! The [Clever Tykes storybooks](https://clevertykes.com/) and resources aim to inspire children to develop positive, resourceful and creative behaviour. The series of four storybooks - Walk-it Willow, Code-it Cody, Change-it Cho and Write-it Ryan - follow characters who turn something they enjoy doing into a venture of their own. That's how we enable children ages 6-9 to experience the entrepreneurial process and encourage them to think and behave in a more enterprising way.
+
+Of specific interest to Exercism's members is the second in the series and by far our most popular, [Code-it Cody](https://clevertykes.com/code-it-cody/), based on a computer whizz who starts coding at school.
+
+**We'll come back to Code-it Cody, but first—what gave you the idea for the series?**
+
+Sure. After noticing that many entrepreneurs came from families of other entrepreneurs, we began to research the impact of role models. Turns out, role models are integral in shaping someone's aspirations, and especially influential in their future choice of career. We looked at the role models that exist in current media and weren't satisfied with the selection, so we decided to write our own!
+
+**I love that it led to the creation of new role models to inspire children. So tell me a little about the idea behind Code-it Cody?**
+
+We see Code-it Cody as being how we inspire children to want to learn more about coding, regardless of their parents' knowledge or what their school happens to teach. There are several tools available that teach children how to code. Many of the parents of budding coders will know of a few. However, we don't want it to only be the children with parents already tuned in to the world of programming to be the ones that go on to be great programmers. Code-it Cody gives readers the inspiration to begin.
+
+**Why did you decide that coding was such an important skill to base one of the stories around?**
+
+We felt it important to give the stories modern context, so technology plays a role across the series. When thinking about essential skills for the future of work, coding was naturally one of the first we thought of. It's well-documented that there is a growing skills gap in this area and coding is not yet embraced by all schools.
+
+There are very few barriers to entry for kids to take their interest in coding further. They can pretty much start learning at any time and this kind of accessibility was really important to us.
+
+**This is right up our street! What about the other characters? How did you choose which topics to focus on and were there any you wrote but didn't release?**
```

Comment:
> and were there any you wrote but didn't release

I've removed this bit and the Bake-it-Benny bit because I don't think it'll be of interest to our readers.